README.md

```diff
@@ -46,13 +46,16 @@ configuration for Ruby's Capybara:
 ```
 chrome_bin = ENV.fetch('GOOGLE_CHROME_SHIM', nil)
```

Comment:
I'd vote ` chrome_bin` was renamed to `chrome_shim` to make it clearer when its used